docs/grid_exchange_formats/cgmes/export.md

```diff
@@ -1,8 +1,154 @@
+
+If the IIDM network has at least one voltage level with node/breaker topology level, and the SSH or SV is requested in the export, and the TP is not requested, an error will be logged, as there could be missing references in the SSH, SV files to Topological Nodes calculated automatically by IIDM that are not present in the output.
+
+If the dependencies have to be updated automatically (see parameter **iidm.export.cgmes.update-dependencies** below), the exported instance files will contain metadata models where:
+* TP and SSH depend on EQ.
+* SV depends on TP and SSH.
+* EQ depends on EQ_BD (if present). EQ_BD is the profile for the boundary equipment definitions.
+* SV depends on TP_BD (if present). TP_BD is the profile for the boundary topology. Only for CGMES 2.4.
+
+The output filenames will follow the pattern `<baseName>_<profile>.xml`. The basename is determined from the parameters, or the basename of the export data source or the main network name. 
+
+## CGM (Common Grid Model) quick export
+
+When exporting a CGM, we need an IIDM network (CGM) that contains multiple subnetworks (one for each IGM). 
+Only the CGMES instance files corresponding to SSH and SV profiles are exported:
+an updated SSH file for every subnetwork (for every IGM) and a single SV file for the main network that represents the CGM.
+
+When exporting, it is verified that the main network and all subnetworks have the same scenario time (network case date). If they are different, an error is logged.
+
+If no version is provided as a parameter for the exported files, the output version is determined based on the max version of input CGM SV metadata and IGM SSH versions.
+
+If the dependencies have to be updated automatically (see parameter **iidm.export.cgmes.update-dependencies** below), the exported instance files will contain metadata models where:
+* Updated SSH for IGMs supersede the original ones.
+* Updated SV for the CGM depends on the updated SSH from IGMs and on the original TP from IGMs.
+
+The filenames of the exported instance files will follow the pattern:
+* For the CGM SV: `<basename>_SV.xml`.
+* For the IGM SSHs: `<basename>_<IGM name>_SSH.xml`. The IGM name is built from the country code of the first substation or the IIDM name if no country is present.
+
+The basename is determined from the parameters, or the basename of the export data source or the main network name.
+
+As an example, you can export one of the test configurations that has been provided by ENTSO-E. It is available in the cgmes-conformity module of the powsybl-core repository. If you run the following code:
+
+```java
+Network cgmNetwork = Network.read(CgmesConformity1Catalog.microGridBaseCaseAssembled().dataSource());
+
+Properties exportParams = new Properties();
+exportParams.put(CgmesExport.PROFILES, List.of("SV", "SSH"));
```

Comment:
What happens here if I put "EQ", "TP", "SSH" and "SV" ?

Reply:
Indeed, after a small test, changing this list does not change anything. We always export SV and SSH, even if I only put SV.

Reply:
clarified the documentation to explicitly state that CGM quick export ignores the `profiles` parameter and always exports SSH IGMs and CGM SV. Removed the parameter from the code example in the docs and from the main unit test showing its use.

Reply:
Have you pushed?